strands_navigation_msgs/msg/Edge.msg

```diff
@@ -6,9 +6,8 @@ string map_2d
 bool use_default_recovery
 string[] monitor_recovery_pairs_names
 strands_navigation_msgs/DynClassLoaderDef[] monitor_states
-strands_navigation_msgs/DynClassLoaderDef[] recovery_state_machines
 bool use_default_nav_recovery
```

Comment:
yeah, if you think it's better I don't mind. If you do change, maybe we can also remove the bools, and assume default to be true iff the yaml string does not have an entry regarding that particular recover